.github/workflows/goreleaser.yml

```diff
@@ -1,21 +1,25 @@
-name: Release
+name: goreleaser
 
 on:
+  workflow_dispatch:
   push:
     tags:
-      - 'v[0-9]+.[0-9]+.[0-9]+'
+      - '**'
```

Comment:
If this is a regex, shouldn't it be `.*`?

Reply:
The doc train I followed from https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-of-specific-tags-occurs -> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore -> https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet made me